common/src/main/java/whocraft/tardis_refined/client/renderer/blockentity/device/ConsoleConfigurationRenderer.java

```diff
@@ -52,7 +52,7 @@ public void render(ConsoleConfigurationBlockEntity blockEntity, float partialTic
             ResourceLocation theme = blockEntity.theme();
 
             ConsoleUnit consoleModel = ConsoleModelCollection.getInstance().getConsoleModel(theme);
-            consoleModel.renderConsole(null, blockEntity.getLevel(), poseStack, bufferSource.getBuffer(RenderType.entityTranslucent(consoleModel.getDefaultTexture())), packedLight, OverlayTexture.NO_OVERLAY, 1f, 0.64f, 0f, 0.5f);
+            consoleModel.renderConsole(null, blockEntity.getLevel(), poseStack, bufferSource.getBuffer(RenderType.entityTranslucent(consoleModel.getDefaultTexture())), packedLight, OverlayTexture.NO_OVERLAY, ( 162f / 255f ), ( 100f / 255f), (224f /255f), 0.5f);
```

Comment:
Personally, I would find the result of these sums and put the result instead of the sum, saves on performance a little bit

Reply:
Turned into a sum.